src/main/java/se/sundsvall/messaging/api/model/response/UserMessage.java

```diff
@@ -0,0 +1,60 @@
+package se.sundsvall.messaging.api.model.response;
+
+import static io.swagger.v3.oas.annotations.media.Schema.AccessMode.READ_ONLY;
+
+import java.time.LocalDateTime;
+import java.util.List;
+
+import io.swagger.v3.oas.annotations.media.ArraySchema;
+import io.swagger.v3.oas.annotations.media.Schema;
+import lombok.Builder;
+import lombok.With;
+
+@Builder(setterPrefix = "with")
+@Schema(description = "User message model")
+public record UserMessage(
+
+	@Schema(description = "The message id", example = "b971e0f8-2942-4b45-9fa3-bd2cc22ed76b")
+	String messageId,
+
+	@Schema(description = "The message issuer", example = "and06sod")
+	String issuer,
+
+	@Schema(description = "The system that the message originated from", example = "CASEDATA")
+	String origin,
+
+	@Schema(description = "When the message was sent", example = "2021-01-01T12:00:00")
+	LocalDateTime sent,
+
+	@ArraySchema(schema = @Schema(implementation = Recipient.class, accessMode = READ_ONLY))
+	List<Recipient> recipients,
+
+	@ArraySchema(schema = @Schema(implementation = MessageAttachment.class, accessMode = READ_ONLY))
+	List<MessageAttachment> attachments) {
+
+	@With
+	@Builder(setterPrefix = "with")
+	@Schema(description = "Recipient model")
+	public record Recipient(
+
+		@Schema(description = "The person identifier", example = "199001011234")
+		String personId,
```

Comment:
Could the recipient be an organization as well (or is it always a private person)? 
I.e.: should this attribute be "partyId" instead?

Reply:
The story specifies that we should return personal number, but I'll double check this with Per.